api/db/activity/index.js

```diff
@@ -12,6 +13,7 @@ module.exports = () => ({
   ...activity,
   ...approach,
   ...contractorResource,
+  ...costAllocation,
```

Comment:
All the data models have to be processed by `/db/index.js` to get their names registered.  The activity models get wrapped up by `/db/activities/index.js` and passed up all in one rather than the db index having to get them all itself.  To get to the point, the model has to be required into one of these two files so the names registered and Bookshelf can resolve the names into objects later.

I realize now that we probably need more documentation around how this works, but I'm not sure where that should be.  Should it be comments in code?  That seems better than putting it a md file or wiki, from my personal dev perspective.  But I'm not sure where in the code it should go.  🤔